src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -377,7 +390,14 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
       selectedEditionSet,
     } = this.state
 
-    const artworkEcommerceAvailable = !!(isAcquireable || isOfferable)
+    const editionSelectableOnInquireable = !!(
+      artwork.is_inquireable && labFeatureEnabled
```

Comment:
If the ff is on shouldn't we display the radios independently if it's "Buy now", "Make Offer" or "Contact Gallery"? (I'm considering that have edition sets)

Reply:
I would say yes if we are completely sure that those are the only states possible (`acquirable, `offerable` and `inquireable`). I just tried to be a bit more conservative here because it is not yet completely clear to me if there is any other possibility that I'm not seeing (and the scenario that we are sure that we're focusing on the MOOAEA project would be the inquireable scenario).  I expect to have a better picture of all possible scenarios during [NX-3057](https://artsyproduct.atlassian.net/jira/software/c/projects/NX/boards/41?modal=detail&selectedIssue=NX-3057).